hack/update-codegen.sh

```diff
@@ -22,7 +22,7 @@ ANTREA_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
 IMAGE_NAME="antrea/codegen:kubernetes-1.21.0"
 
 function docker_run() {
-  docker pull ${IMAGE_NAME}
+  #docker pull ${IMAGE_NAME}
```

Comment:
comment by mistake?

Reply:
I am using a local image for codegen with new verison of protoc-gen-go. Therefore, commenting this when running codegen on local machine. Once the image on antrea repo on docker registry is updated (before merging this PR), I will remove this.